packages/rainbowkit/src/wallets/walletConnectors/infinityWallet/infinityWallet.ts

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { openInfinityWallet } from '@infinitywallet/infinity-connector';
```

Comment:
Since this is using wagmi's injected connector, can we avoid adding another dependency?

Reply:
Hi @markdalgleish, as Infinity Wallet is not a browser extension but the first standalone web3 browser wallet, the function `openInfinityWallet` is used both to detect if Infinity Wallet is installed on click and to open the DApp in the Infinity Wallet from any browser.

If you would like, rather than importing the dependency for this we could directly integrate the code into the repository?

Is there anything else required to merge this?